File: plugin-test/src/main/java/me/mattstudios/mfmsg/TestPlugin.java
package me.mattstudios.mfmsg;

import me.mattstudios.mf.base.CommandManager;
import org.bukkit.plugin.java.JavaPlugin;

public final class TestPlugin extends JavaPlugin {

    @Override
    public void onEnable() {
        final CommandManager commandManager = new CommandManager(this);
        commandManager.register(new Cmd());
    }

}

File: plugin-test/src/main/java/me/mattstudios/mfmsg/TestPlugin.kt
package me.mattstudios.mfmsg

import org.bukkit.plugin.java.JavaPlugin
import me.mattstudios.mf.base.CommandManager
import me.mattstudios.mfmsg.Cmd

class TestPlugin : JavaPlugin() {

    override fun onEnable() {
        val commandManager = CommandManager(this)
        commandManager.register(Cmd(this))
    }

}